<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Discard Changes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QuestionDialog {
	background-image: url(:/img/pattern_dark.png);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <property name="rightMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Discard Changes?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>You've changed some values of the current movie. If you don't save them now, your changes will be lost.
Would you like to continue and lose your changes or go back and save them?</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="MediaElch.qrc">:/img/warning1.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonDiscard">
       <property name="text">
        <string>Discard Changes</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="text">
        <string>Go back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
